packages/api/src/schema/db-schema.yaml

```diff
@@ -587,12 +587,9 @@ components:
       table: webhook
       properties:
         kind:
-          type: string
-          example: webhook
           readOnly: true
```

Comment:
This is not a custom field AFAIK. It exists in OpenAPI spec: https://swagger.io/docs/specification/data-models/data-types/#readonly-writeonly

If we're moving the field to public docs (api-schema), this prop should also go with it. It is part of the spec/definition.

Same for many other fields below, won't repeat the comment.

Reply:
sounds good, will add readOnly/writeOnly to api-schema as well